lib/views/rules/_list_of_rules.html.erb

```diff
@@ -0,0 +1,24 @@
+  <div class="read-container js-read-container">
+    <p class="read-container__announcement"><strong>Please read WhatDoTheyKnow rules before continue:</strong></p>
+    <ol>
```

Comment:
I think we should extract the list of rules in [`help/house_rules.html.erb`](https://github.com/mysociety/whatdotheyknow-theme/blob/75f8614e61a23bb12a706904da0f22bfb38a0785/lib/views/help/house_rules.html.erb#L7-L31) to a partial (`help/house_rules/_list_of_house_rules.html.erb`), and then render _that_ partial here.

This saves us from having to maintain two versions of the rules.